Automattic/pocket-casts-ios
# Update the listening time story to use a scaled size for the covers

| 📘 Project: #374 | 🛫 Depends on: #562 |
|:---:|:---:|

This changes the podcast covers on the listening time (story 2) to fill more space on all screen sizes. 

## Screenshots

### Before / After (14 pro max)
| Before | After |
|:---:|:---:|
|<img src="https://user-images.githubusercontent.com/793774/205421224-33694c03-35a3-4ed2-87dd-eb39f56afd3c.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205421279-dd5cb5cb-29c7-4fe8-afb0-91932f00b054.png" width="320" />|


### Devices
| iPhone SE - 14.1 | iPhone 13 | iPhone 14 Pro Max |
|:---:|:---:|:---:|
|<img src="https://user-images.githubusercontent.com/793774/205421331-0f0b8237-df5b-4d5c-934e-1460d4a2a7a4.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205421298-eacfe83f-18c4-420d-8594-c3d2ab3b04ed.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205421279-dd5cb5cb-29c7-4fe8-afb0-91932f00b054.png" width="320" />|


## To test

1. Launch the app
2. Tap the Profile Tab > End of Year Card
3. Switch to the second story
4. ✅ Verify it looks good 
5. ✅ Verify the assets are shared
6. Repeat the steps on various screen sizes

## Checklist
